### ISSUE TITLE
Work on original homepage (html & bootstrap)

### ISSUE THREAD

**Issue by PumpkinKing93** [CSCI-150-SCRUM/SCRUM-150#6]
Create a modular homepage that will be able to access new boards, new users, boards.. etc. 

upload to github, with the index.html call to run it all.

**Comment by kristenlabrie** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Kristen's test site</title>
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
	<script src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
	<link href="style.css" rel="stylesheet">
</head>
<body>
<div style="background-image: url('background.jpg');"></div>



<div class="container-fluid">
	
	<div clas="collapse navbar-collapse" id="navbarResponsive">
			<ul class="navbar-nav ml-auto">
				<li class="nav-item">
					<a class="nav-link" href="#">Existing Projects</a>
				</li>
				<li class="nav-item">
						<a class="nav-link" href="#">Add a team member</a>
				</li>
				<li class="nav-item">
						<a class="nav-link" href="#">Add a manager</a>
				</li>
			</ul>
		</div>
</div>
</nav>

<font color="white"> START A NEW PROJECT</font>

<div class = "container text-center">
		<button type="button-link">+</button>
</div>


</body>
</html>

@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,700');




body{
  background-image: url('background.jpg');

}
a {
  color: white;
  font-family: Comic Sans Ms, Comic Sans, cursive;
}
button{
  background-color: rgb(236, 182, 82);
  font-size: 40px;
  color: white;
  border-radius: 50%;
  width: 100px;
  height: 100px;
  border: 2px solid white;
  margin: 0 auto;
}
font{
  font-family: Comic Sans Ms, Comic Sans, cursive;
}




/*---Media Queries --*/
@media (max-width: 992px) {

  }
@media (max-width: 768px) {
  
}
@media (max-width: 576px) {
  
}


/*---Firefox Bug Fix --*/
.carousel-item {
  transition: -webkit-transform 0.5s ease;
  transition: transform 0.5s ease;
  transition: transform 0.5s ease, -webkit-transform 0.5s ease;
  -webkit-backface-visibility: visible;
  backface-visibility: visible;
}
/*--- Fixed Background Image --*/
figure {
  position: relative;
  width: 100%;
  height: 60%;
  margin: 0!important;
}
.fixed-wrap {
  clip: rect(0, auto, auto, 0);
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}
#fixed {
  background-image: url('img/mac.png');
  position: fixed;
  display: block;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-size: cover;
  background-position: center center;
  -webkit-transform: translateZ(0);
          transform: translateZ(0);
  will-change: transform;
}
/*--- Bootstrap Padding Fix --*/
[class*="col-"] {
    padding: 1rem;
}





/*
Extra small (xs) devices (portrait phones, less than 576px)
No media query since this is the default in Bootstrap

Small (sm) devices (landscape phones, 576px and up)
@media (min-width: 576px) { ... }

Medium (md) devices (tablets, 768px and up)
@media (min-width: 768px) { ... }

Large (lg) devices (desktops, 992px and up)
@media (min-width: 992px) { ... }

Extra (xl) large devices (large desktops, 1200px and up)
@media (min-width: 1200px) { ... }
*/
